godtools/App/Features/AppLanguage/Data/DownloadedLanguagesRepository/Cache/RealmDownloadedLanguagesCache.swift

```diff
@@ -26,10 +26,17 @@ class RealmDownloadedLanguagesCache {
             .eraseToAnyPublisher()
     }
     
-    func getDownloadedLanguagesPublisher() -> AnyPublisher<[DownloadedLanguageDataModel], Never> {
+    func getDownloadedLanguagesPublisher(completedDownloadsOnly: Bool) -> AnyPublisher<[DownloadedLanguageDataModel], Never> {
         
-        let downloadedLanguages = realmDatabase.openRealm()
+        var realmDownloadedLanguages = realmDatabase.openRealm()
             .objects(RealmDownloadedLanguage.self)
+        
+        if completedDownloadsOnly {
+            realmDownloadedLanguages = realmDownloadedLanguages
+                .where { $0.downloadComplete }
+        }
```

Comment:
I'm not sure if this is what you meant or not 😄 but it's a little better since it returns the same Result type.  Is there a different way to do it that I'm missing?

Originally I was doing the condition inside the filter, like
```
var downloadedLanguages = realmDatabase.openRealm()
            .objects(RealmDownloadedLanguage.self)
            .filter { completedDownloadsOnly ? $0.downloadComplete : true }
            .map { DownloadedLanguageDataModel(realmDownloadedLanguage: $0) }
```
But thought there's no reason to perform the filter when `completedDownloadsOnly = false` just because it looks cleaner.  I guess `.where` would also work in that ^ scenario.  Seems like the only difference between `.filter` and `.where` is the return type.  And maybe it's more readable to use the word "where" in cases like this.

Reply:
Hey @rachaelblue oh wow my eyes and my brain were seeing something different this morning.  

For some reason I thought your else statement was filtering on objects where downloadComplete was false.  But, if that were the case you could still use ```.filter``` and check if downloadComplete was equal to completedDownloadsOnly without the need for the conditional.

Sorry for the confusion there!  I was seeing something different and my brain was thinking something different.